templates/todo/api/java-postgresql/src/main/java/com/microsoft/azure/simpletodo/api/ApiUtil.java

```diff
@@ -1,7 +1,7 @@
 package com.microsoft.azure.simpletodo.api;
 
-import java.io.IOException;
 import jakarta.servlet.http.HttpServletResponse;
+import java.io.IOException;
```

Comment:
Why are there java changes in this PR.  shouldn't it be bicep only changes?

Reply:
@jongio Reverted, please review again.